library/func_misc.py

```diff
@@ -206,7 +206,7 @@ def Call(self, args):
 
         strs = []  # type: List[str]
         for i, el in enumerate(li):
-            strs.append(val_ops.Stringify(el, loc.Missing))
+            strs.append(val_ops.Stringify(el, args.BlameLoc()))
```

Comment:
ditto here, does it make sense to pass  `(` directly ?

Like maybe `args.ParenToken()`

Then it's clearer that we're not being more precise than that, which is fine